Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM python:3.11-bullseye
```

Comment:
If possible, I recommend using [`3.11-slim-bullseye`](https://hub.docker.com/layers/library/python/3.11-slim-bullseye/images/sha256-cd6e574ff6ad2ac75184ad90e4e374842cfc36ae21bc2f773531ac65f6939991?context=explore) instead which shaves off a ton in image size.

Reply:
will be looked at another time, atm breaks dkp-pacman install script